docs/api/app.md

```diff
@@ -1184,6 +1184,14 @@ Show the app's about panel options. These options can be overridden with `app.se
 Set the about panel options. This will override the values defined in the app's
 `.plist` file on MacOS. See the [Apple docs][about-panel-options] for more details. On Linux, values must be set in order to be shown; there are no defaults.
 
+### `app.showEmojiPanel` _macOS_ _Windows_
+
+Show the platform's native emoji picker.
+
+### `app.showColorPanel` _macOS_
+
+Show the platform's native color picker.
```

Comment:
How do you get the chosen color here?

Also this can be implemented on Windows --> https://msdn.microsoft.com/en-us/library/ms646912(v=VS.85).aspx